src/transbank_webpay_rest/library/HealthCheck.php

```diff
@@ -26,61 +26,62 @@ public function __construct($config)
         $this->commerceCode = $config['COMMERCE_CODE'];
         $this->apiKey = $config['API_KEY'];
         $this->ecommerce = $config['ECOMMERCE'];
-        // extensiones necesarias
         $this->extensions = [
             'openssl',
             'SimpleXML',
             'dom',
         ];
     }
 
-    // valida version de php
+    /**
+     * Validates the current PHP version.
+     *
+     * @return array The status and the current PHP version.
+     */
     private function getValidatephp()
     {
-        if (version_compare(phpversion(), '7.4', '<=') and version_compare(phpversion(), '7.0.0', '>=')) {
-            $this->versioninfo = [
-                'status'  => 'OK',
-                'version' => phpversion(),
-            ];
-        } else {
-            $this->versioninfo = [
-                'status'  => 'Error!: Versión no soportada',
-                'version' => phpversion(),
-            ];
-        }
+        $minVersion = '7.0.0';
+        $maxVersion = '7.4.0';
+        $currentVersion = phpversion();
+        $isValidVersion = version_compare($currentVersion, $minVersion, '>=') && version_compare($currentVersion, $maxVersion, '<=');
+        $this->versioninfo = [
```

Comment:
is necessary set this `versionInfo` as an attribute? or can return this directly